client/src/pages/guide/english/apache/index.md

```diff
@@ -4,5 +4,20 @@ title: Apache
 ## Apache
 
 The Apache HTTP Server, commonly known as Apache, is a free and open-source cross-platform web server, released under the terms of [Apache License 2.0](https://en.wikipedia.org/wiki/Apache_License). Apache is developed and maintained by an open community of developers under the auspices of the Apache Software Foundation.
+Apache runs on 67% of all webservers in the world. It is fast, reliable, and secure. It can be highly customized to meet the needs of many different environments by using extensions and modules
 
 
+### Installation
```

Comment:
Like in the other PR, I don't think this section adds any value. Only a subset of platforms and incomplete. Official docs are always better for this.

Reply:
Even though the docs may be more complete, it doesn't hurt to have this section imho.